transposon/genotypeClones.py

```diff
@@ -557,6 +573,8 @@ def jaccard(G, u, v):
     parser.add_argument("--removeConflictingCells", action='store_true', default=False, help="Removes cells linked to BCs from 2+ transfections. It requires `transfectionKey`")
     parser.add_argument("--removeConflictingBCs", action='store_true', default=False, help="Removes BCs linked to cells from 2+ transfections. It requires `transfectionKey`")
     
+    parser.add_argument("--minjaccard", action="store", type=float, default=0, help="Unlink cells with Jaccard-index below this threshold")
```

Comment:
How about "Remove edges connecting cells that whose BC overlap is below this Jaccard-index threshold"

Reply:
"Remove edges connecting cells whose BC overlap is below this Jaccard-index threshold" sounds good?

Reply:
perfect